<?xml version="1.0" encoding="UTF-8"?><testsuite name="nosetests" tests="1" errors="0" failures="1" skip="0"><testcase classname="test_unittest.Test_Search_Module" name="test_search_index" time="0.001"><failure type="builtins.AssertionError" message="None != 1&#10;-------------------- &gt;&gt; begin captured stdout &lt;&lt; ---------------------&#10;binary search: 9 found at index 5&#10;elapsed time: 3.058000000000227e-06 seconds&#10;halvings: 3 &#10;&#10;--------------------- &gt;&gt; end captured stdout &lt;&lt; ----------------------"><![CDATA[  File "/Users/jthompson/opt/anaconda3/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Users/jthompson/opt/anaconda3/lib/python3.7/unittest/case.py", line 628, in run
    testMethod()
  File "/Users/jthompson/dev/PythonFundamentals.Exercises.Algos/test_unittest.py", line 8, in test_search_index
    self.assertEqual(sm.binary_search([24, 10, 1, 64, 99, 9, 12], 9), 1)
  File "/Users/jthompson/opt/anaconda3/lib/python3.7/unittest/case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/Users/jthompson/opt/anaconda3/lib/python3.7/unittest/case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
None != 1
-------------------- >> begin captured stdout << ---------------------
binary search: 9 found at index 5
elapsed time: 3.058000000000227e-06 seconds
halvings: 3 

--------------------- >> end captured stdout << ----------------------]]></failure><system-out><![CDATA[binary search: 9 found at index 5
elapsed time: 3.058000000000227e-06 seconds
halvings: 3 
]]></system-out></testcase></testsuite>